utils/common_utils.py

```diff
@@ -69,6 +65,42 @@ def get_term_descendants(ontology, term_url, logger):
             logger.error("Failed to receive valid response from {}.".format(api_url))
 
 
+def get_ena_library_terms_via_usi(logger):
+    """Read ENA's controlled vocabulary using USI's API and
+    return dictionary of the field names that have a set of allowed values (enum)."""
+
+    url = "https://submission-dev.ebi.ac.uk/api/dataTypes/sequencingExperiments"
```

Comment:
For URLs, you could have a small optional config file that could be provided to the tool as part of its cli. If the URLs are unlikely to change, that plus defaults in the code would be a reasonable approach (I think, in the end these are all things of style).